.swiftlint.yml

```diff
@@ -181,7 +181,7 @@ unused_optional_binding:
   ignore_optional_try: false
 
 file_length:
-  warning: 500
+  warning: 550
```

Comment:
By adding an additional unit test to `CoreTests`, the test extended the file length past 500.

Reply:
Not sure if just updating our linter rules every time it inconveniences us is the best approach. I don't want to block on this specifically - but we may want to silence the warning, disable swift lint in tests, or something else.

@wessmith thoughts?